### PR TITLE
OCPBUGS#13837: Improve the note

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-13837

Link to docs preview: https://63690--docspreview.netlify.app/openshift-enterprise/latest/security/certificate_types_descriptions/bootstrap-certificates.html#expiration

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: security/certificate_types_descriptions/bootstrap-certificates.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ The kubelet-managed certificate is valid for one year and rotates automatically
 
 [NOTE]
 ====
-OpenShift Lifecycle Manager (OLM) does not update the certificates of Operators that it manages in proxy environments. These certificates must be managed by the user using the subscription config.
+OpenShift Lifecycle Manager (OLM) does not update the Bootstrap CA. The Bootstrap CA must be managed by the user using the subscription config.
```
Comment (jianzhangbjz) [inline security/certificate_types_descriptions/bootstrap-certificates.adoc:26]: > The Bootstrap CA must be managed by the user using the subscription config.

What's the `the subscription config` means here? OLM managed subscription doesn't manage the `Bootstrap CA`.

Comment (xenolinux) [reply security/certificate_types_descriptions/bootstrap-certificates.adoc]: @jianzhangbjz I updated the PR after discussing it with the bug reporter. Can you PTAL?

Comment (jianzhangbjz) [reply security/certificate_types_descriptions/bootstrap-certificates.adoc]: Sorry, I missed it. LGTM.

**Review by abhatt-rh — COMMENTED** [review]
Hi @xenolinux Left a ques and a suggestion. Thanks!

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: security/certificate_types_descriptions/bootstrap-certificates.adoc [hunk 2]
```diff
@@ -23,7 +23,7 @@ The kubelet-managed certificate is valid for one year and rotates automatically
 
 [NOTE]
 ====
-OpenShift Lifecycle Manager (OLM) does not update the certificates of Operators that it manages in proxy environments. These certificates must be managed by the user using the subscription config.
+OpenShift Lifecycle Manager (OLM) does not update the Bootstrap CA.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply security/certificate_types_descriptions/bootstrap-certificates.adoc]: Ack'ed

**Pull request merged**